deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -561,6 +561,7 @@ def _create_regular_windows(path, name, content=''):
     """
     regular_path = os.path.join(path, name)
     os.popen("echo " + content + " > " + regular_path + f" runas /user:{os.getlogin()}")
+    time.sleep(3)
```

Comment:
Deleted the function _create_regular and _create_regular_windows because they  were not used in the repository.
There are further functions to remove.

Fixed in https://github.com/wazuh/wazuh-qa/pull/2802/commits/585afaabb989d41944cf5ff0353bfa6f4a3bf74f

Reply:
Deleted the timeout. it does not affect the creation of the file.